packages/block-editor/src/components/inserter/style.scss

```diff
@@ -274,14 +274,14 @@ $block-inserter-tabs-height: 44px;
 	}
 }
 
-.block-editor-inserter__quick-inserter-results {
-	padding-bottom: $grid-unit-20;
+.block-editor-inserter__quick-inserter-results .block-editor-inserter__panel-header {
+	height: 0;
+	padding: 0;
+	float: left;
+}
 
-	.block-editor-inserter__panel-header {
-		height: 0;
-		padding: 0;
-		float: left;
-	}
+.block-editor-inserter__quick-inserter:not(.has-search):not(.has-expand) .block-editor-inserter__panel-content {
```

Comment:
For future-proofing - say at some point we add another class in addition to search and expand, for another variation, and then we're stuck with overriding this super-specific rule - could we not add the wider padding to `.has-search` and `.has-expand` instead, and make this one the default?

Reply:
Good suggestion! Fixed in dc0a9fdfc1d9e7abf7836adf9cfdb8593e98eb21.